{
	"id": "CVE-2025-1628",
	"sourceIdentifier": "cve@gitlab.com",
	"vulnStatus": "Rejected",
	"published": "2025-03-19T23:15:12.783",
	"lastModified": "2025-03-19T23:15:12.783",
	"descriptions": [
		{
			"lang": "en",
			"value": "Rejected reason: This CVE ID has been rejected or withdrawn by its CVE Numbering Authority."
		}
	],
	"references": [],
	"metrics": {}
}
